tests/rbd_mirror/rbd_mirror_utils.py

```diff
@@ -887,3 +896,151 @@ def mirror_daemon_status(self, daemon):
         if daemon_status.strip("\n") != "active":
             return 1
         return 0
+
+
+def rbd_mirror_config(**kw):
+    """
+    Configure RBD mirroring on replicated pools, ecpools or both based on arguments specified
+    Args:
+        **kw:
+
+    Examples: In default configuration, pool and image names will be taken as random values.
+        Default configuration for ecpools only :
+            config:
+                ec-pool-only: True
+        Default configuration for replicated pools only :
+            config:
+                rep-pool-only: True
+        Advanced configuration:
+            config:
+               do_not_create_image: True  # if not set then images will be created by default
+               ec-pool-k-m: 2,1
+               ec-pool-only: False
+               ec_pool_config:
+                  pool: rbd_pool_4
+                  data_pool: rbd_ec_pool_4
+                  ec_profile: rbd_ec_profile_4
+                  image: rbd_image_4
+                  size: 10G
+                  io_total: 1G
+                  mode: image (default pool)
+                  mirrormode: snapshot (default journal)
+               rep_pool_config:
+                  pool: rbd_rep_pool
+                  image: rbd_rep_image
+                  size: 10G
+                  io_total: 1G
+                  mode: image (default pool)
+                  mirrormode: snapshot (default journal)
+    Args:
+        **kw:
+
+    Returns:
+        RBD mirror objects for ecpool and replicated pool
+    """
+    rbd_obj = dict()
+
+    if not kw.get("config") or not kw.get("config").get("ec-pool-only"):
```

Comment:
can you please provide `if else` condition scenario in comments ?

Reply:
Added comment as discussed